onnx/__init__.py

```diff
@@ -24,8 +25,13 @@ def load(obj):
     '''
     model = ModelProto()
     if hasattr(obj, 'read') and callable(obj.read):
-        model.ParseFromString(obj.read())
```

Comment:
I know few about python, looks you may want to use decoded = model.ParseFromString(obj.read()) :)